packages/gestalt/src/Sticky.js

```diff
@@ -20,16 +20,17 @@ type Threshold =
     |};
 
 type Props = {|
+  ...Threshold,
   children: Node,
-  zIndex?: Indexable,
   dangerouslySetZIndex?: {| __zIndex: number |},
-  ...Threshold,
+  height?: number,
```

Comment:
@jackhsu978 should we also make this a number or string? Like we do on Box?

![image](https://user-images.githubusercontent.com/127199/89693055-356fdd00-d8c2-11ea-9c0d-486e9648d1ad.png)